### PR TITLE
Ported ApiCompat

### PR DESCRIPTION
**Pull request by venkat-raman251**
@weshaggard : Please have a look

### CODE REVIEW

File: src/ApiCompat/Mappings.txt [hunk 1]
```diff
@@ -0,0 +1,79 @@
+﻿# Mappings file for Silverlight to Jupiter UI stacks.
```
Comment (weshaggard) [inline src/ApiCompat/Mappings.txt]: This file isn't needed it was only part of an experiment. Please delete it.

File: src/ApiCompat/NamespaceRemappingComparer.cs [hunk 2]
```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```
Comment (weshaggard) [inline src/ApiCompat/NamespaceRemappingComparer.cs]: We should update all the copyrights to match what @stephentoub is doing right now. https://github.com/dotnet/buildtools/pull/418

**Comment by weshaggard** [comment]
Do you have a way to show the diff of what you had to change to make it .NET Core complaint?

**Comment by venkat-raman251** [comment]
@weshaggard : I have updated the licenses and also created an initial commit (with the code as it previously was) so that you can see the diff.

**Comment by weshaggard** [comment]
The more I think about how to integrate this the more I think it will be best to actually create a wrapper msbuild task for ApiCompat that we can hock into the individual project builds. So after we get the source ported then please play with creating a msbuild task that wraps this.

**Comment by venkat-raman251** [comment]
@weshaggard : Fixed most of the problems that you had pointed out. Please let me know if it looks ok

**Comment by weshaggard** [comment]
What about all my comments/questions directly on commit https://github.com/venkat-raman251/buildtools/commit/b95a2b39467c352d4c4f0e664cc24d2637481a27

**Comment by venkat-raman251** [comment]
I fixed them based off your comments on that commit (CLScompliance,license,assemblyInfo and some renaming).
The questions you had were about a couple of try-catch blocks which didn't make any sense. Must have missed 'em while cleaning up the code. So i removed it.

**Comment by venkat-raman251** [comment]
@weshaggard - Removed AssemblyInfo, Added handling for ExitCode using a static variable.

**Comment by venkat-raman251** [comment]
@weshaggard  Could you please have a look.

**Comment by weshaggard** [comment]
LGTM thanks @venkat-raman251

**Pull request merged**